seatunnel-connectors-v2/connector-hudi/src/main/java/org/apache/seatunnel/connectors/seatunnel/hudi/util/HudiUtil.java

```diff
@@ -68,14 +69,15 @@ public static String getParquetFileByPath(String confPaths, String path) throws
         return null;
     }
 
-    public static SeaTunnelRowType getSeaTunnelRowTypeInfo(String confPaths, String path) throws HudiPluginException {
+    public static SeaTunnelRowType getSeaTunnelRowTypeInfo(String confPaths, String path) throws HudiConnectorException {
         Configuration configuration = getConfiguration(confPaths);
         Path dstDir = new Path(path);
         ParquetMetadata footer;
         try {
             footer = ParquetFileReader.readFooter(configuration, dstDir, NO_FILTER);
         } catch (IOException e) {
-            throw new HudiPluginException("Create ParquetMetadata Fail!", e);
+            throw new HudiConnectorException(HudiConnectorErrorCode.CREATE_PARQUET_METADATA_FAILED,
```

Comment:
CommonErrorCode.TABLE_SCHEMA_GET_FAILED is better.

Reply:
Done, Thank you for your reminder